src/components/share-overlay/share-overlay.js

```diff
@@ -64,7 +64,7 @@ const ShareButton = (props: Object): React$Element<any> => {
         window.open(href, '_blank', 'width=580,height=580');
         break;
     }
-    props.player.dispatchEvent(new FakeEvent(ShareEvent.SHARE_NETWORK, {shareNetworkName: buttonType}));
+    props.player.dispatchEvent(new FakeEvent(ShareEvent.SHARE_EMBED_SHARE_VIA_SOCIAL_NETWORK, {socialNetworkName: buttonType}));
```

Comment:
I don't see this event in the latest list in the ticket. Is it missing from the list ?